packages/postgres-database/src/simcore_postgres_database/models/users.py

```diff
@@ -77,8 +81,23 @@ class UserStatus(str, Enum):
         "name",
         sa.String(),
         nullable=False,
-        doc="username is a unique short user friendly identifier e.g. pcrespov, sanderegg, GitHK, ...",
+        doc="username is a unique short user friendly identifier e.g. pcrespov, sanderegg, GitHK, ..."
```

Comment:
name or username? or nickname or pseudo?

Reply:
Will follow up in https://github.com/ITISFoundation/osparc-simcore/issues/6913